src/bookshelf/rebar.lock

```diff
@@ -51,19 +65,26 @@
   {git,"https://github.com/davisp/jiffy",
        {ref,"ee8b48cb448a999e1fddbb5e2e32e7dfe33acf5c"}},
   1},
- {<<"jsx">>,{pkg,<<"jsx">>,<<"2.9.0">>},1},
+% {<<"jsx">>,{pkg,<<"jsx">>,<<"2.9.0">>},1},
+ {<<"jsx">>,
+  {git,"https://github.com/talentdeficit/jsx",
+       {tag,"2.9.0"}},
+  1},
  {<<"lager">>,
   {git,"https://github.com/erlang-lager/lager",
        {ref,"a140ea935eae9149bb35234bb40f6acf1c69caa1"}},
   0},
- {<<"lhttpc">>,{pkg,<<"lhttpc">>,<<"1.6.2">>},1},
+ {<<"lhttpc">>,
+  {git,"https://github.com/erlcloud/lhttpc",
+       {ref,"8e34985a3cd0ac2a7fc2a88a041554c64d33e74b"}},
+  1},
  {<<"meck">>,
   {git,"https://github.com/eproxus/meck",
        {ref,"06192a984750070ace33b60a492ca27ec9bc6806"}},
   0},
  {<<"mini_s3">>,
   {git,"https://github.com/chef/mini_s3",
-       {ref,"8a6b80ec60be799c5f69d9b2f22eb0af55b4dc77"}},
+       {branch,"lbaker/upgrade-erlang-24.2"}},
```

Comment:
Would it be possible to change it to reference/tag instead of branch name.

Reply:
@sreepuramsudheer The mini_s3 changes have not been merged yet, that's why we are referencing the special branch.